docs/tasks.md

```diff
@@ -1,5 +1,21 @@
 # What can it do?
 
+List of all the keys under `config.tasks`:
+
+| Name                                                                                       | Key           |
+| ------------------------------------------------------------------------------------------ | ------------- |
+| [Add Repositories](#add-repositories)                                                      | `key`         |
+| [Add personal package archives (PPA)](#add-personal-package-archives-ppa)                  | `ppa`         |
+| [Install apt packages](#install-apt-packages)                                              | `apt`         |
+| [Install Debian package archives (.deb files)](#install-debian-package-archives-deb-files) | `dev`         |
+| [Install Static binaries to /usr/local/bin](#install-static-binaries-to-usrlocalbin)       | `binaries`    |
+| [Install python packages (via pip)](#install-python-packages-via-pip)                      | `pip2`/`pip3` |
+| [Purge Unwanted Packages](#purge-unwanted-packages)                                        | `purge`       |
+| [Reset repositories / Purge PPAs](#reset-repositories-purge-ppas)                          |               |
```

Comment:
Reset function will be removed in v 7.0 and above. This is done due to deprecation of apt-key. Remove this line.

Reply:
Do you mean keep this line but remove the link to "Reset all repos"?

Reply:
Ah my bad! yes. reset will be removed in v7.0

Reply:
It was my mistake, I wrote on mobile.

It is confusing, I created this PR compared to the deployed docs, but now I see the deployed version is outdated compared to the `master`.

Anyway, I removed it.